mitodl/micromasters
# Display coupon-discounted prices in UI

#### What are the relevant tickets?
Fixes #2403 

#### What's this PR do?
Adds logic to contact the coupons API to get the list of coupons, and calculates adjusted prices for each course run.

#### How should this be manually tested?
Create a coupon locally and verify that it adjusts the price of the 'Pay now' button
